mrjones2014/yaclt
# Matjones/151 enhanced readme

Resolves: #151 
Resolves: #152 
Resolves: #153 

## How to Test

1. Review changes to `README.md`
2. Run `yarn build`
3. Run `mv yacltrc.js yacltrc.js.tmp` to temporarily wipe out the config
4. Run `./dist/index.js validate` -- should get error about missing required options
